### ISSUE TITLE
Allow showing multiple samples (ie trio)

### ISSUE THREAD

**Issue by anderspitman** [iobio/vue-pileup#13]

**Comment by anderspitman** [comment]
@tonydisera I have a prototype of this working. What do you think of [this API](
https://github.com/anderspitman/vue-pileup/blob/allow-multiple-tracks/src/App.vue) for showing multiple tracks? Basically just passing it a list of objects containing the URLs

**Comment by tonydisera** [comment]
Excellent!  I think the only other field the 'tracks' entry will need is sample id.   Actually, how is this currently being handled?  The samples will often be in the same vcf, distinguisable by sample id.

**Comment by anderspitman** [comment]
Oh interesting, I didn't realize that. As far as I know it simply shows all variants in the VCF. Let me see if there's a way to specify. Do you happen to know if normal igv has that functionality?

**Comment by tonydisera** [comment]
I see.  So the alignment track is specific to the sample, but the variant track is across all samples.  This will be fine for multi-sample vcfs that only have the trio under analysis, but often there will be background samples in the vcf.  Is there a way to send in sample ids to the igv plugin?

<img width="1112" alt="screen shot 2018-11-14 at 2 02 43 pm" src="https://user-images.githubusercontent.com/8420103/48512209-2572b880-e816-11e8-923d-fd2a8a72606d.png">

**Comment by anderspitman** [comment]
Yeah I don't think there's a way to do it, but I'll open an issue on igv.js just in case

**Comment by tonydisera** [comment]
Yeah, I like the way the samples show in separate tracks.  But we will also want to hide samples from the track.  


<img width="1294" alt="screen shot 2018-11-14 at 2 13 28 pm" src="https://user-images.githubusercontent.com/8420103/48512827-a41c2580-e817-11e8-97aa-207e847b6fec.png">

**Comment by anderspitman** [comment]
[Here's the issue](https://github.com/igvteam/igv.js/issues/796)

**Comment by anderspitman** [comment]
Do we know what sample ids we're interested in?

**Comment by tonydisera** [comment]
Yes, we will be able to pass that into your component.

**Comment by anderspitman** [comment]
ok, I'll keep you posted on what they say upstream

**Comment by anderspitman** [comment]
@tonydisera, @jrobinso was kind enough to offer to implement the variant functionality upstream to allow us to only show variants for the sample ids we want. I'll go ahead and keep this issue open until we have a complete solution. But I'm going to go ahead and merge what I have so far so that we can start testing the multitrack stuff in gene. You should be able to bump to version 0.4.0 to get that functionality.

**Comment by tonydisera** [comment]
Excellent!
